UI/cypress/e2e/full-list.js

```diff
@@ -99,15 +99,14 @@ Then(
 );
 
 Then("the name of each tuition partner links to their details page", () => {
```

Comment:
May not be so easy to do and may have already considered it - any new cypress tests needed?

Reply:
I see what I can do about this👍